server/routes/unusedDeductionRoutes.ts

```diff
@@ -75,6 +80,45 @@ export default class UnusedDeductionRoutes {
     const { prisonerNumber } = res.locals.prisoner
     const { username } = res.locals.user
     const { bookingId } = res.locals.prisoner
+    const reviewDeductions = this.paramStoreService.get(req, 'returnToReviewDeductions')
+    if (reviewDeductions) {
+      const adjustments = await this.getAdjustments(bookingId, username, nomsId)
+      const unusedDeductionAdjustment =
+        this.getNomisUnusedDeduction(adjustments) || this.getUnusedDeduction(adjustments)
+      const sessionAdjustmentRecords = this.adjustmentsStoreService.getAll(req, nomsId)
+      const sessionAdjustments: SessionAdjustment[] = Object.keys(sessionAdjustmentRecords).map(
+        key => ({ ...sessionAdjustmentRecords[key] }) as SessionAdjustment,
+      )
+
+      const sentencesAndOffences = await this.prisonerService.getSentencesAndOffences(bookingId, username)
+      const unusedDeductionsResponse = await this.calculateReleaseDatesService.unusedDeductionsHandlingCRDError(
+        sessionAdjustmentRecords,
+        adjustments,
+        sentencesAndOffences,
+        nomsId,
+        username,
+      )
+
+      adjustments
+        .filter(it => it.adjustmentType === 'REMAND' || it.adjustmentType === 'TAGGED_BAIL')
+        .forEach(it => {
+          if (!sessionAdjustments.find(adj => adj.id === it.id)) {
+            sessionAdjustments.push(it)
+          }
+        })
+
+      return res.render('pages/adjustments/unused-deductions/review', {
+        model: new UnusedDeductionsReviewModel(
+          prisonerNumber,
+          unusedDeductionAdjustment ? 'edit' : 'add',
+          unusedDeductionsResponse.unusedDeductions || 0,
+          sessionAdjustments.filter(it => !it.delete),
+          [],
+          reviewDeductions,
+        ),
+      })
+    }
+
     const adjustments = await this.getAdjustments(bookingId, username, nomsId)
```

Comment:
Can you split this into a separate endpoint? Seems odd to have MUD + RUD go through the same endpoints.